indico/modules/events/registration/controllers/api/checkin.py

```diff
@@ -130,3 +133,33 @@ def _process_args(self):
 
     def _process_GET(self):
         return CheckinRegistrationSchema().jsonify(self.registration)
+
+
+class RHCheckinAPIRegistrationCustomQRCode(RHCheckinAPIBase):
+    """Get indico style ticket details for a specific registration from a custom plugin QR Code."""
+
+    def _process_args(self):
+        pass
+
+    def _check_access(self):
+        pass
+
+    @use_kwargs({
+        'data': fields.String(required=True),
+        'name': fields.String(required=True),
```

Comment:
I guess that name would be defined by the plugin, and be something like `un-access-badge`, `cern-access-badge`, etc.?

Reply:
Yes, that's correct